### DISCUSSION TITLE
Can I get bright without bold (as in a console)?

### DISCUSSION THREAD

**Discussion by vefatica** [microsoft/terminal#12114] [Q&A] [answered]
Microsoft Windows 10 Pro for Workstations
10.0.19042.1110 (2009, 20H2)
WindowsTerminal: 1.10.1933.0

Below, a Windows console is on top and in the foreground, WindowsTerminal on the bottom and in the background.  This is the best size-match I can get ... Consolas 22 Normal in the console, Consolas 17 Normal in WT.  Apparently, when I specify the bright/bold attribute in WT, I get both bright (a different color) and bold (a different font weight).  In the console I get just the bright color.

Can I get bright without bold in WT?

![image](https://user-images.githubusercontent.com/61856645/126679914-290d53fd-f8a8-4475-af68-baaa9921929b.png)

I think something has changed recently.  I've been looking at the bright green "v:\>" prompt for ages, and only very recently has the fuzzy bold version of it bothered me.

**Comment by zadjii-msft** [comment, marked as answer]
> I think something has changed recently.

yep, we added support for bold text in #10498. 

Adding the setting to control this is in #10576, and the PR should be #10759 (unless someone snipes me).

/dup #10576

**Comment by ghost** [comment]
Hi! We've identified this issue as a duplicate of another one that already exists on this Issue Tracker. This specific instance is being closed in favor of tracking the concern over on the referenced thread. Thanks for your report!

**Comment by vefatica** [comment]
Here are the 1.9 preview (first) and the 1.10 preview (second) using the same settings file.

![image](https://user-images.githubusercontent.com/61856645/126682224-f7d1dc23-5c05-487e-87cb-7e9d6cd20b96.png)

![image](https://user-images.githubusercontent.com/61856645/126682344-1f3c5212-5854-4aad-929c-8e16024b62b5.png)

**Comment by vefatica** [comment]
> yep, we added support for bold text in #10498

You not only added support for it, you made it the default, with no way to get the behavior to which users are accustomed, both in WT and the Windows console.

**Comment by DHowett** [comment]
We have to move forward. I'm not interested in making Terminal start up and act just like conhost in every single way.

**Comment by DHowett** [comment]
As well: Preview is a preview. We release features there to gauge how users will receive them :smile:. We declared _before even merging support for bold fonts_ that it needed to be configurable before we would take it out of the Preview version and into Stable.

**Comment by vefatica** [comment]
I just changed my app alias to this (and I get this).  Notice how the message box specifies the preview.

![image](https://user-images.githubusercontent.com/61856645/126686902-8cabfa37-bcec-40ed-bcb3-475e1cfd3d6c.png)

**Comment by vefatica** [comment]
In the post above, I issued "WT" in the Win+R  dialog.  These are OK and both work correctly.

![image](https://user-images.githubusercontent.com/61856645/126687771-aa186a35-73dc-4c75-a5f6-60286ba39a08.png)

**Comment by vefatica** [comment]
So I uninstalled both.  The preview would re-install but the release version gives this.

![image](https://user-images.githubusercontent.com/61856645/126695386-12b7295b-dc10-4fa6-9638-3b9268d10fc2.png)
 The unshown part of the reason is:
`
S-1-5-18 (0x80070005)`

Got any hints on fixing this?

**Comment by vefatica** [comment]
After a restart, I could (re-)install both versions (release and preview).  But I'm still seeing peculiar behavior from the App Alias settings.

If I toggle these:

![image](https://user-images.githubusercontent.com/61856645/126708385-cddc608b-6fe3-4a97-8915-4b30569a090b.png)

the Settings app remembers my choice but **there are no changes in these registry keys**:

```
2021-07-22 16:44:46.296  HKCU\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\wt.exe
2021-07-22 16:44:41.597  HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\wt.exe
```

If I manually change those keys (using .reg files) I can use the "WT" alias successfully if and only if what's in those keys matches the App Alias choice.  If they don't match, I get the "access denied" message (regarding the one specified in the registry).
